### PR TITLE
Utils folders duplicity fixed

### PR DESCRIPTION
**Pull request by Josejpm**
# Summary
 Utils folder was duplicated, creating issues in windows users, it was fixed putting all utils methods inside Utils folder and updating all the import of those files

# Changes Added
- utils folder deleted
- files in utils folder moved to Utils
- imports updated in order to clear errors and get the project on air

### CODE REVIEW

File: src/Components/Activities/ActivitiesForm.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Alert } from '@mui/material';
 import { CKEditor } from '@ckeditor/ckeditor5-react';
 import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
 import { useDropzone } from 'react-dropzone';
-import { dropzoneConfig, isEmptyList } from '../../Utils/ActivitiesUtils';
+import { dropzoneConfig, isEmptyList } from '../../Utils';
```
Comment (saralavanchy) [inline src/Components/Activities/ActivitiesForm.js:12]: are this functions on Utils?

Comment (saralavanchy) [reply src/Components/Activities/ActivitiesForm.js:12]: it looks like isEmptyList is not

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Activities/ActivitiesForm.js (hunk 1 above)
Comment (Josejpm) [reply src/Components/Activities/ActivitiesForm.js:12]: yes, they are, both methods are in Utils/index.js

Comment (Josejpm) [reply src/Components/Activities/ActivitiesForm.js:12]: This is the actual branch screenshot

image

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**